Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace VRTK
+{
+	using UnityEngine;
+	using System.Collections;
+
+	public class VRTK_Hip_Tracking : MonoBehaviour {
+
+		/// <summary>
+    	/// Hip Tracking attempts to reasonably track hip position in the absence of a hip position sensor
+    	/// </summary>
+   	 	/// <remarks>
+		///
+		/// The Hip Tracking script is placed on an empty GameObject which will be positioned at the estimated hip position
+		/// <para>&#160;</para>
+		/// Variables
+		/// <list type="bullet">
+    	/// <item>Head Offset<description>Distance underneath Player Head for hips to reside.</description> </item>
+		/// <item>Head Override<description>Optional Transform to use as the Head Object for calculating hip position. If none is given one will try to be found in the scene.</description> </item>
+		/// <item>Reference Up<description>Optional Transform to use for calculating which way is "Up" relative to the player for hip positioning.</description> </item>
+    	/// </list>
+		/// </remarks>
+
+		Transform playerHead;
```

Comment:
need to determine the accessibility of the parameter (and if it's private it needs to go below the public then protected params)